### DISCUSSION TITLE
Allow variables for dashboardList filtering

### DISCUSSION THREAD

**Discussion by KnpA** [grafana/grafana#38276] [Dashboards, Panels & Visualizations]
**What would you like to be added**: 

Use variables to filter dashboardList panel results with **tags** and "**query**" in search options.

An example with a textbox variable, but could also work with constant, custom or query variables:
![image](https://user-images.githubusercontent.com/6367927/117122254-ff4e8380-ad95-11eb-8ff9-37e96c6ffd5c.png)

**Why is this needed**:

Our users don't use Grafana main menu to access dashboards and prefer homepages with subsets like this:
![image](https://user-images.githubusercontent.com/6367927/117123497-63257c00-ad97-11eb-9459-476132ceff0c.png)

In this example, we would like to use a device variable, obtain his vendor and filter all dashboardLists to get only dashboards associated with this vendor (and thus that device).

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
